vendor/ember-cli-qunit/test-container-styles.css

```diff
@@ -1,33 +1,53 @@
+#qunit {
+  position: absolute;
+  top: 0;
+  right: 0;
+  height: 100%;
+  overflow: scroll;
+  width: 33%;
+  background-color: white;
+  word-wrap: break-word;
+  margin: 0;
+  padding: 0;
+}
+#qunit-header {
+  border-radius: 0;
+}
+.qunit-filter {
+  display: block;
+  float: none;
+  margin-left: 0;
+  padding: 0.2em;
+}
+#qunit-modulefilter-container {
+  float: none;
+}
+#qunit-modulefilter {
+  width: 100%;
+}
+
 #ember-testing-container {
-  position: relative;
+  position: absolute;
+  left: 0;
+  top: 0;
+  background: inherit;
+  border: none;
+  height: 100%;
+  width: 66%;
+  z-index: 0;
+  margin: 0;
   background: white;
-  bottom: 0;
-  right: 0;
-  width: 640px;
-  height: 384px;
   overflow: auto;
   z-index: 9999;
-  border: 1px solid #ccc;
-  margin: 0 auto;
 }
 
 #ember-testing-container.full-screen {
   width: 100%;
   height: 100%;
+  top: 0;
+  left: 0;
```

Comment:
These `top` and `left` values also seem unneeded since they're set in `#ember-testing-container` as well.